src/styles/getMuiTheme.js

```diff
@@ -12,6 +13,8 @@ import zIndex from './zIndex';
  * theme will be deeply merged with the second argument.
  */
 export default function getMuiTheme(baseTheme, muiTheme) {
+  if (process.env.NODE_ENV !== 'production') checkTapEventPlugin();
```

Comment:
I have tested this on my project. Unfortunately, `getMuiTheme` is called before `injectTapEventPlugin`.
That won't work.
What about moving the check to the function that apply the style?